packages/agoric-cli/src/commands/vaults.js

```diff
@@ -2,28 +2,61 @@
 /* eslint-disable @jessie.js/no-nested-await */
 // @ts-check
 /* eslint-disable func-names */
-/* global fetch */
+/* global fetch, process */
+import { vstorageKeySpecToPath } from '@agoric/casting';
 import { Command } from 'commander';
+import { normalizeAddressWithOptions } from '../lib/chain.js';
 import { makeRpcUtils } from '../lib/rpc.js';
 import { makeOpenSpendAction } from '../lib/vaults.js';
-import { outputAction } from '../lib/wallet.js';
+import { getCurrent, outputAction } from '../lib/wallet.js';
 
 /** @typedef {import('@agoric/smart-wallet/src/offers').OfferSpec} OfferSpec */
 /** @typedef {import('@agoric/smart-wallet/src/offers').OfferStatus} OfferStatus */
 /** @typedef {import('@agoric/smart-wallet/src/smartWallet').BridgeAction} BridgeAction */
 
-const {
-  vstorage,
-  fromBoard: _fromBoard,
-  agoricNames,
-} = await makeRpcUtils({ fetch });
+const { vstorage, fromBoard, agoricNames } = await makeRpcUtils({ fetch });
 
 /**
  *
  * @param {import('anylogger').Logger} logger
  */
 export const makeVaultsCommand = async logger => {
-  const vaults = new Command('vaults').description('Vault Factory commands');
+  const vaults = new Command('vaults')
+    .description('Vault Factory commands')
+    .option('--home [dir]', 'agd application home directory')
+    .option(
+      '--keyring-backend [os|file|test]',
+      'keyring\'s backend (os|file|test) (default "os")',
+    );
```

Comment:
why keyring options? we're not signing, are we?

ah... we're looking up addresses by name. got it.